substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2823,6 +2849,58 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Apply a pending slash on a member.
+		///
+		/// This call can be dispatched permissionlessly (i.e. by any account). If the member has
+		/// slash to be applied, caller may be rewarded with the part of the slash.
```

Comment:
Would be great to write a simple PJS/python bot for this (you can add it to `polkadot-scripts`). And we should simulate slashes in westend.

Reply:
Yes absolutely. We should have this bot running all the time as well. Since it gets part of the slash as reward, we should probably see other bots from the community as well.

Reply:
I have mentioned this in the followup issues, will create a separate issue for it at some point.